openapi-cli/src/main/java/io/ballerina/openapi/generators/client/BallerinaClientGenerator.java

```diff
@@ -306,6 +325,63 @@ private FunctionDefinitionNode createInitFunction() throws BallerinaOpenApiExcep
      * @return  {@link FunctionBodyNode}
      */
     private FunctionBodyNode getInitFunctionBodyNode() {
+        List<StatementNode> assignmentNodes = new ArrayList<>();
+
+        // If both apiKey and httpOrOAuth is supported
+        if (ballerinaAuthConfigGenerator.isApiKey() && ballerinaAuthConfigGenerator.isHttpOROAuth()) {
```

Comment:
Shall we move this logic of init statement generation at mixed authentication scenario to a new function to increase the readability of the code? WDYT :)

Reply:
Done